static/scripts/experience.js

```diff
@@ -1,23 +1,86 @@
-const $ = window.$;
+//const $ = window.$;
 
+let stagedExp = [];
```

Comment:
@dbconfession78  needs to moved back into $(document).ready

Reply:
done